about.html

```diff
@@ -0,0 +1,92 @@
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a class="campaign" href="#">CC Campaign</a></li>
+            </ul>
+        </div>
+        <a href="#" class="toggle-btn">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </a>
+        <div id="hamburger-menu">
+            <span class="close-btn">&times;</span>
+            <ul class="hamburger-menu">
+                <li><a href="about.html">About</a></li>
+                <li><a href="index.html">Home</a></li>
+            </ul>
+      </div>
+    </header>
+    <section id="home-about">
+            <div id="about-us">
+              <!-- <h3 class="hello">"Hello! Sharing world"</h3> -->
+              <h1>Udemy Online Courses 2023</h1>
+              <div class="underline"></div>
+              <p class="about-course">The Udemy Courses 2023 brings together the community of experts, academics, and activists who teach web development in a different country every two years. This year, we hope to expand our invitation list including orgaizations and individuals</p>
+              <p class="about-course-dskt">The Udemy Courses 2023 brings together the community of experts, academics, and activists who teach web development in a different country every two years. This year, we hope to expand our invitation list including orgaizations and individualsThe Udemy Courses 2023 brings together the community of experts, academics, and activists who teach web development in a different country every two years. This year, we hope to expand our invitation list including orgaizations and individuals</p>
+              <p class="contact">Please contact us per Email for any further questions about Udemy online course 2023!   <br><strong>udemy2023@udsanfransisco.org</strong>
+              </p>
+           </div>
+
+        <div id="about-logo">
+          <h3>Udemy Online Course 2023 Logo</h3>
+          <div class="underline"></div>
+          <p class="logo-mob">The logo of Udemy online course 2023 was decided through the logo competetion from 3.May to 5.June.</p>
+          <p class="logo-dskt">The logo of Udemy online course 2023 was decided through the logo competetion from 3.May to 5.June. After online voting selection and committe's process, the logo from Bob Jackson won the prize. After consultation with him this following logo was optimized</p>
+        </div>
+        <div id="about-time">
+          <img src="images/logo2.png" alt="udemy course's logo">
+        </div>
+
+        <div id="past-courses">
+          <h3>See the past Udemy Online Courses</h3>
+          <div class="underline"></div>
+          <p>Take a look at the last two Udemy Online Courses which took place in Dushanbe and New York.</p>
+          <img src="images/dushanbe.jpg" alt="past course in Dushanbe">
+          <img src="images/newYork.jpg" alt="past course in New York">
```

Comment:
Kindly add overlay color and text to the images like on the design guideline.
<img width="885" alt="Screenshot 2023-02-09 at 9 22 12 PM" src="https://user-images.githubusercontent.com/25887253/217903716-f7c230d5-88b4-498a-8281-6a2fe5a203ca.png">

<img width="1440" alt="Screenshot 2023-02-09 at 9 23 12 PM" src="https://user-images.githubusercontent.com/25887253/217903521-d85a4679-7850-49cc-9bc8-61e27f2a6405.png">